### PR TITLE
Query Browser style fixes and allow passing buttonClassName and menuClassName to <Dropdown />

### PR DESCRIPTION
**Pull request by kyoto**
Fix dropdown style issues introduced by switching to PF4 dropdowns. Required allowing `buttonClassName` and `menuClassName` to be passed to `Dropdown`.

Several other Query Browser style tweaks.

FYI @sg00dwin

### CODE REVIEW

File: frontend/public/components/graphs/_graphs.scss [hunk 1]
```diff
@@ -56,44 +56,39 @@
...
   width: 20px;
 }
 
 .query-browser__metrics-dropdown {
-  width: 200px;
-}
-.query-browser__metrics-dropdown-menu {
-  max-height: 25vh;
-  overflow: scroll;
+  .query-browser__metrics-dropdown-button {
```
Comment (spadgett) [inline frontend/public/components/graphs/_graphs.scss]: Avoid nesting when possible with BEM

Comment (kyoto) [reply frontend/public/components/graphs/_graphs.scss]: Nesting removed by https://github.com/openshift/console/pull/1881 and https://github.com/openshift/console/pull/1896

**Pull request merged**